github/data_source_github_ip_ranges.go

```diff
@@ -0,0 +1,53 @@
+package github
+
+import (
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceGithubIpRanges() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceGithubIpRangesRead,
+
+		Schema: map[string]*schema.Schema{
+			"hooks": &schema.Schema{
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+			"git": &schema.Schema{
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+			"pages": &schema.Schema{
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+		},
+	}
+}
+
+func dataSourceGithubIpRangesRead(d *schema.ResourceData, meta interface{}) error {
+	org := meta.(*Organization)
+
+	api, _, err := org.client.APIMeta(org.StopContext)
+	if err != nil {
+		return err
+	}
+
+	if len(api.Hooks)+len(api.Git)+len(api.Pages) > 0 {
+		d.SetId("github-ip-ranges")
```

Comment:
I don't know what to set here? I suppose it doesn't matter in this case? Anyone let me know!